src/index.d.ts

```diff
@@ -0,0 +1,5 @@
+import fs from 'fs';
+import stream from 'stream';
+
+export function toObject(xmlStream: fs.ReadStream, schema: string?, objectPath: string[], { strict: boolean = false, trimText: boolean = true, ignoreTagNameSpace: boolean = false } = {}): stream.Readable;
```

Comment:
Could you change the `fs.ReadStream` to `stream.Readable`? It is more generic and the implementation doesn't care if the read stream comes from `fs` or somewhere else. Same with the function below.